packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -189,7 +189,9 @@ export function RepositoryMixin<T extends MixinTarget<Application>>(
       nameOrOptions?: string | BindingFromClassOptions,
     ) {
       const binding = super.component(componentCtor, nameOrOptions);
-      this.mountComponentRepositories(componentCtor);
+      const instance = this.getComponentInstance(componentCtor);
```

Comment:
We should use `binding.key` to get the component instance.

Reply:
Good point 👍